app.py

```diff
@@ -215,11 +212,12 @@ def create_folder_section(folder_number, section_name, db_get):
     section_object = db_get(folder_number)
     action = "add"
     status = "To be filled"
+    last_modified_on = datetime.datetime.today().strftime('%Y-%m-%d')
```

Comment:
Make into util fn?

Reply:
can. I put it in as a placeholder till the section is actually added.

Reply:
also would like to date and time (strftime('%Y-%m-%d-%H-%M')) but then not sure what to do when putting into dbase. the datetime.combine line.